recipes/discasm/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+DISCASM_INSTALL_DIR="$PREFIX/share/DISCASM-0.1.2-0"
```

Comment:
If you want, you can literally use `$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM`, as @daler suggested. Then you wouldn't need to change it when you update the recipe to a later version.
Those `$PKG_` variables are available at build time, see https://conda.io/docs/user-guide/tasks/build-packages/environment-variables.html.

Reply:
In case it's useful, here's how to resolve the symlink (so it looks like it's the right directory) in python. https://github.com/bioconda/bioconda-recipes/blob/07aa987a0c9ebe7975bd66afd783026431b86883/recipes/busco/generate-busco-config.py#L25, which avoids hard-coded paths.

Reply:
Cool. Thanks.

Reply:
I just scanned through the code to see if that is done. It is done for some, but not all situations. Not sure why Brian did use realpath some places, but not others. If it gets changed in the future, I will be able to change the shell scripts to symlinks.